### PR TITLE
feat(tax-rate)!: Add tax rate

### PR DESCRIPTION
**Pull request by JumpLink**
Add the tax rate to the frontend

grafik

Backend: https://github.com/beabee-communityrm/beabee/pull/397

### CODE REVIEW

**Review by wpf500 — CHANGES_REQUESTED** [review]
Can you move the tax rate setting from the membership builder into the settings? It can just go here under a new heading ("Payment"?)

image

**Comment by JumpLink** [comment]
> Can you move the tax rate setting from the membership builder into the settings? It can just go here under a new heading ("Payment"?)

sure

**Comment by JumpLink** [comment]
@wpf500 I have moved the tax settings to the general settings page [on this branch](https://github.com/beabee-communityrm/beabee-frontend/pull/579). I think now the data should also be stored there in the database and no longer in join content, or what do you think?

File: src/pages/admin/settings/index.vue [hunk 1]
```diff
@@ -87,6 +87,47 @@ meta:
...
+          </div>
+        </div>
+        <div class="mb-4 flex gap-4">
+          <div class="flex-1">
+            <AppLabel :label="t('adminSettings.payment.taxRate')" />
+            <div class="max-w-[8rem]">
+              <AppInput
+                v-model="paymentData.taxRate"
+                type="number"
+                :min="1"
```
Comment (wpf500) [inline src/pages/admin/settings/index.vue]: I think 0% could be a legit tax rate

File: src/pages/admin/settings/index.vue [hunk 2]
```diff
@@ -87,6 +87,47 @@ meta:
...
+                :max="100"
+                :disabled="!paymentData.taxRateEnabled"
+                prefix="%"
```
Comment (wpf500) [inline src/pages/admin/settings/index.vue]: `suffix` better than `prefix` ?

**Review by wpf500 — COMMENTED** [review]
I think this PR doesn't add the tax rate to the join form right? Just the admin side?

**Comment by JumpLink** [comment]
> I think this PR doesn't add the tax rate to the join form right? Just the admin side?

Yes, I think the join part can be implemented together with new UX design, or should I add the tax also to the current to be able to ship this without the new join UX flow?

**Review by wpf500 — APPROVED** [review]

**Pull request merged**